src/Lykke.Service.Pledges.Core/Settings/ServiceSettings/ServiceSettings.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Lykke.Service.Pledges.Core.Settings.ServiceSettings
+{
+    public class ServiceSettings
+    {
+        public string ClientAccountServiceUrl { get; set; }
```

Comment:
use it as a separate class like :
public class ClientAccountService
{
   public string ServiceUrl {get;set;}
}
and use it as a property in AppSettings:
public ClientAccountService ClientAccountClient {get;set;}

so you'll end up with the json settings like this (a top level property for the client account service) - http://13.95.227.175/e7b6cb180ad24cb3bf4c84974f96872a_LykkeRegistration
and will use access token ClientAccountClient.* on the service settings

Reply:
Done